neqo-http3/src/hframe.rs

```diff
@@ -37,14 +38,116 @@ pub enum HStreamType {
     Push,
 }
 
-#[derive(PartialEq, Debug)]
+#[derive(Clone, PartialEq, Debug, Copy)]
 pub enum HSettingType {
     MaxHeaderListSize,
-    MaxTableSize,
+    MaxTableCapacity,
     BlockedStreams,
     UnknownType,
 }
 
+fn hsetting_default(setting_type: HSettingType) -> u64 {
```

Comment:
I think that there is enough setting-specific code here that settings can be moved to a new file.